servers.go

```diff
@@ -24,6 +24,13 @@ type Server struct {
 	Platform   string `json:"platform"`
 }
 
+//Are used to create user(s) on new machine
```

Comment:
Jag tror att best practice (required?) är att börja kommentaren med namnet på typen...

```suggestion
// User represents a system user when creating servers (currently supported in KVM)
```